addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2107,6 +2107,13 @@ const SnippetOptionWidget = Widget.extend({
             await this._select(previewMode, widget);
             this.$target.trigger('content_changed');
 
+            // Enabling an option and notifying that the $target has changed
+            // may destroy the option (if the DOM is altered in such a way the
+            // option is not attached to it anymore). In that case, we must not
+            // wait for a response to the option update.
+            if (this.isDestroyed()) {
+                return;
+            }
```

Comment:
@bvr-odoo This will fix the gallery problem :+1:

Reply:
Ok thanks a lot